OpenShift/helm/nexus-iq/templates/ingress.yaml

```diff
@@ -26,22 +28,18 @@ spec:
     {{- end }}
   {{- end }}
   rules:
-  {{- range .Values.ingress.hosts }}
-    - host: {{ . | quote }}
+    - host: iq-server.demo
       http:
         paths:
           - path: {{ $ingressPath }}
             backend:
               serviceName: {{ $fullName }}
-              servicePort: http
-  {{- end }}
-  {{- range .Values.service.additionalPorts }}
-    - host: {{ .host | quote }}
+              servicePort: 8070
+    - host: admin.iq-server.demo
       http:
         paths:
           - path: {{ $ingressPath }}
             backend:
               serviceName: {{ $fullName }}
-              servicePort: {{ .name }}
-  {{- end }}
+              servicePort: 8071
```

Comment:
This is configurable via `iq.adminPort` so we should be using that rather than hardcoding it.